docs/about/contributing.rst

```diff
@@ -0,0 +1,31 @@
+.. _contributing:
+
+Contributing to xcipp
+=====================
+
+Overview
+~~~~~~~~
+
+Contributions, bug reports, and ideas are always welcome.
+The following section outlines the scope of scipp.
+If in doubt whether a feature falls within the scope of scipp please `ask on github <https://github.com/scipp/scipp/issues>`_ before implementing functionality, to reduce the risk of rejected pull requests.
+Asking and discussing first is generally always a good idea, since our road map is not very mature at this point.
+
+Scope
+-----
+
+While scipp is in practice developed for neutron-data reduction, the scipp library itself must be kept generic.
```

Comment:
Suggestion: `funded and developed for`.

Suggestion: `must be kept generic because at its core it is not neutron specific and has much wider application`.